code/modules/trade/datums/trade_stations_presets/caduceus.dm

```diff
@@ -0,0 +1,67 @@
+/datum/trade_station/caduceus
+	name_pool = list("SAV 'Caduceus'" = "old Sol republic Aid Vessel 'Caduceus'. They sending message, \"Hello, there, we are from Old Sol Republic, we will soon jump back, cuz we get what we wanted, and while we here we can offer medical help for you.\"")
```

Comment:
They're sending a message. Hello there, we are from the Old Sol Republic. We will be leaving the system shortly but we can offer you any medical aid while we are still here.